en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -76,12 +76,17 @@ Although the design is well built such that you can do this later as well.
 
 ### Power
 
-- The Pixhawk 6C gets powered by a power module PM02 (in this case).
-- This power module is supplied by a battery (4S 16.8V 5200 mAh)
-- The motors get power through the power distribution board.
+The Pixhawk 6C gets powered by a power module PM02 (in this case).
+This power module is supplied by a battery (4S 16.8V 5200 mAh)
```

Comment:
When you split the docs up like this it is obvious that we don't have an image of the power module stuff. If you have one it would be good to add. I'll merge this, but if you can add it later that would b egood.